collectors/ebpf.plugin/ebpf.c

```diff
@@ -27,8 +27,10 @@ struct config collector_config = { .first_section = NULL,
 int running_on_kernel = 0;
 int ebpf_nprocs;
 int isrh = 0;
+int main_thread_id = 0;
 
 pthread_mutex_t lock;
+pthread_mutex_t ebpf_exit_cleanup;
```

Comment:
Hi Thiago!

Should there be a call for this for `pthread_mutex_init` ? (Or I couldn't see?)

Reply:
I was taking a look and I observed that I initialized it in a test branch, but I did not bring it here. Thanks! :man_facepalming: